src/SFA.DAS.EmployerCommitmentsV2.Web/Views/Cohort/Details.cshtml

```diff
@@ -136,31 +153,41 @@
                         </label>
                     </div>
                 }
-
-                @if (Model.ShowApprovalOption)
+                
+                @if (Model.EmployerCanApprove)
                 {
-                    <div class="govuk-radios__item">
-                        <input class="govuk-radios__input" id="radio-approve" name="selection" type="radio" value="@CohortDetailsOptions.Approve" data-aria-controls="conditional-approve">
-                        <label class="govuk-label govuk-radios__label" for="radio-approve">
-                            Yes, approve and notify training provider
-                        </label>
-                    </div>
+                    if (Model.ShowApprovalOption)
+                     {
+                         <div class="govuk-radios__item">
+                             <input class="govuk-radios__input" id="radio-approve" name="selection" type="radio" value="@CohortDetailsOptions.Approve" data-aria-controls="conditional-approve">
+                             <label class="govuk-label govuk-radios__label" for="radio-approve">
+                                 Yes, approve and notify training provider
+                             </label>
+                         </div>
+                     }
+                    if (!Model.ShowApprovalOptionMessage)
+                     {
+                         <div class="govuk-radios__conditional govuk-radios__conditional--hidden" id="conditional-approve">
+                             <div class="govuk-form-group">
+                                 <label class="govuk-label" for="approve-details">
+                                     Leave @Model.ProviderName a message (optional)
+                                 </label>
+                                 <textarea class="govuk-textarea" id="approve-details" name="approvemessage" rows="5"></textarea>
+                             </div>
+                         </div>
+                     }
                 }
-                @if (!Model.ShowApprovalOptionMessage)
+                
+                @{ var requestChangesLabel = new string("No, request changes from training provider");  }
```

Comment:
We can probably do away with the variable altogether, and definitely the "new string" bit at least.

Reply:
Agree with the "new string".

This was the best way  I could think of to change the content of the label based on a conditional, suggestions are welcome.